app/models/methodology.rb

```diff
@@ -154,7 +154,7 @@ def content
     if name_changed?
       doc.xpath('/*/name/text()')[0].replace(@name)
       @content = doc.to_s
-      @changed_attributes.clear
+      changes_applied
```

Comment:
`changes_applied` has the implication of still storing the changes that happened on the record. So it's not quite the same thing as clearing the previous hash. If tests are green seems okay though. Just wanted to point out the difference.

Reply:
I went back in the docs (`4.0`) to see when `changed_attributes.clear` was supposed to be called when using `ActiveModel::Dirty`. It was supposed to be called on `save`: https://github.com/rails/rails/blob/4-0-stable/activemodel/lib/active_model/dirty.rb#L43

In the current version of the docs (`5.2`), on `save` I see the recommended call is `changes_applied`: https://github.com/rails/rails/blob/5-2-stable/activemodel/lib/active_model/dirty.rb#L48

If the sepcs go green, I think we can keep this change.